test/functional/sendheaders.py

```diff
@@ -232,32 +247,37 @@ def run_test(self):
         inv_node.sync_with_ping()
         test_node.sync_with_ping()
 
-        self.test_null_locators(test_node)
+        self.test_null_locators(test_node, inv_node)
         self.test_nonnull_locators(test_node, inv_node)
 
-    def test_null_locators(self, test_node):
+    def test_null_locators(self, test_node, inv_node):
         tip = self.nodes[0].getblockheader(self.nodes[0].generate(1)[0])
         tip_hash = int(tip["hash"], 16)
 
+        inv_node.check_last_announcement(inv=[tip_hash], headers=[])
+        test_node.check_last_announcement(inv=[tip_hash], headers=[])
+
         # TODO this partly fixes the same thing that is fixed by https://github.com/bitcoin/bitcoin/pull/13192
         # This will later conflict when backporting the actual fix. Just take everything from the Bitcoin fix as a
         # resolution
         assert_equal(test_node.check_last_announcement(headers=[], inv=[tip_hash]), True)
 
```

Comment:
This is no longer needed
```suggestion
```

Reply:
Why? (13192 is in my backlog and in that commit I removed this)

Reply:
f798771323ace08b2cba8b7877e2e64b273fd03f added `test_node.check_last_announcement(inv=[tip_hash], headers=[])` right above this which does the the same but also pops from `self.last_message` which causes this line to fail now.